### PR TITLE
Task #224216: change button sequance on delete popup,add condition for no user name found in list, remove unwanted code.

### PR DESCRIPTION
**Pull request by upendraTekdi**

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -82,9 +83,9 @@
     "MAIN_COURSE": "Main Course"
   },
   "FACILITATORS": {
-    "ALL_STATES": "States",
-    "ALL_DISTRICTS": "Districts",
-    "ALL_BLOCKS": "Blocks",
+    "ALL_STATES": "State",
```
Comment (itsvick) [inline public/locales/en/common.json:86]: @upendraTekdi Key should be changed accordingly

File: public/locales/en/common.json [hunk 2]
```diff
@@ -168,7 +169,8 @@
     "CENTER_UPDATE_SUCCESSFULLY": "Center has been Successfully Updated",
     "CENTER_DELETE_SUCCESSFULLY": "Center has been Successfully Deleted",
     "CENTER_CREATED_SUCCESSFULLY": "Center Created Successfully",
-    "SEARCHBAR_PLACEHOLDER": "Search Center.."
+    "SEARCHBAR_PLACEHOLDER": "Search Center..",
+    "NOT_ABLE_CREATE_CENTER": "Not Able To Create Center For Current Selected Data"
```
Comment (itsvick) [inline public/locales/en/common.json:173]: @upendraTekdi keep only first letter capital

File: src/utils/app.constant.ts [hunk 3]
```diff
@@ -57,3 +57,8 @@ export enum Numbers {
   ONE = 1,
   TEN = 10,
 }
+
+export enum Types {
```
Comment (itsvick) [inline src/utils/app.constant.ts:61]: @upendraTekdi rename it to the CohortTypes

**Review by itsvick — CHANGES_REQUESTED** [review]

File: public/locales/en/common.json (hunk 2 above)
Comment (upendraTekdi) [reply public/locales/en/common.json:173]: done

File: public/locales/en/common.json (hunk 1 above)
Comment (upendraTekdi) [reply public/locales/en/common.json:86]: done to 
 "STATE": "State",
    "DISTRICT": "District",
    "BLOCK": "Block",

File: src/utils/app.constant.ts (hunk 3 above)
Comment (upendraTekdi) [reply src/utils/app.constant.ts:61]: done

File: public/locales/en/common.json (hunk 1 above)
Comment (itsvick) [reply public/locales/en/common.json:86]: @upendraTekdi cant see here

**Review by itsvick — APPROVED** [review]

**Pull request merged**